src/main/java/com/blackducksoftware/integration/hub/alert/web/actions/AuditEntryActions.java

```diff
@@ -157,13 +159,15 @@ private AuditEntryRestModel createRestModel(final AuditEntryEntity auditEntryEnt
         NotificationRestModel notificationRestModel = null;
         if (!notifications.isEmpty() && notifications.get(0) != null) {
             try {
-                // TODO check to see the notificationTypes field does not mess up the conversion
                 notificationRestModel = objectTransformer.databaseEntityToConfigRestModel(notifications.get(0), NotificationRestModel.class);
+                final Set<String> notificationTypes = notifications.stream().map(notification -> notification.getNotificationType()).collect(Collectors.toSet());
```

Comment:
This may conflict with my changes. The notificationType field has been changed to an enum.